src/core/client_channel/retry_filter_legacy_call_data.cc

```diff
@@ -115,6 +119,7 @@ RetryFilter::LegacyCallData::CallAttempt::CallAttempt(
     RetryFilter::LegacyCallData* calld, bool is_transparent_retry)
     : RefCounted(GRPC_TRACE_FLAG_ENABLED(retry) ? "CallAttempt" : nullptr),
       calld_(calld),
+      per_attempt_recv_timer_handle_(TaskHandle::kInvalid),
```

Comment:
Please do this initialization in the .h file, where the data member is defined.

Reply:
Done.